Dockerfile_WithSMT

```diff
@@ -0,0 +1,22 @@
+FROM maven:3-openjdk-11 AS build
```

Comment:
I think this file should be `KeyCheckSMT/Dockerfile`

Reply:
I moved this under its own directory https://github.com/ably/kafka-connect-ably/pull/92/commits/de1148ab83730127b76397fb89ad7b61ec2bff8d

Reply:
Resolving this on behalf of @lmars as it was addressed